packages/react-components/react-progress/src/components/Progress/Progress.types.ts

```diff
@@ -1,17 +1,56 @@
 import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
 
 export type ProgressSlots = {
-  root: Slot<'div'>;
+  /**
+   * The root of the Progress
+   * The root slot receives the `className` and `style` specified directly on the `<Progress>`.
+   */
+  root: NonNullable<Slot<'div'>>;
+  /**
+   * The title of the Progress.
+   * The label slot receives the styling related to the title associated with the Progress.
+   */
+  label?: Slot<'span'>;
+  /**
+   * The animated slot of the Progress
+   * The bar slot receives the styling related to the loading bar associated with the Progress
+   */
+  bar?: NonNullable<Slot<'div'>>;
+  /**
+   * The track slot of the Progress
+   * The track slot receives the styling related to the loading bar track associated with the Progress
+   */
+  track?: NonNullable<Slot<'div'>>;
+  /**
+   * The description slot of the Progress
+   * The description slot receives the styling related to the description associated with the Progress
+   */
+  description?: Slot<'span'>;
 };
 
 /**
  * Progress Props
  */
-export type ProgressProps = ComponentProps<ProgressSlots> & {};
+export type ProgressProps = Omit<ComponentProps<ProgressSlots>, 'size'> & {
+  /**
+   * Prop to set whether the Progress is determinate or indeterminate
+   * @default false
+   */
+  indeterminate?: boolean;
+  /**
+   * Percentage of the operation's completeness, numerically between 0 and 1. If this is not set,
+   * the indeterminate progress animation will be shown instead.
```

Comment:
I don't think this last sentence is accurate right?

Also is the value between 0 and 1 or 0 and 100? If it's called _percent_ complete, I'd expect it to be 0-100.

```suggestion
   * Percentage of the operation's completeness, numerically between 0 and 1.
```